pages/comunidade/index.js

```diff
@@ -95,7 +95,7 @@ const Community = ({ credentials }) => {
 Community.getInitialProps = async (ctx) => {
   if (!ctx.query.slug) {
     ctx.res.writeHead(302, {
-      Location: '/',
+      Location: '_error',
     });
```

Comment:
```suggestion
      Location: '/',
```
Aqui e nos outros casos, a gente ta travando 400 ou 401. Pro erro 404 a gente so precisa criar o component _error.js que o next resolve o resto